### PR TITLE
Add release announcement for Bazel 0.29.

### PR DESCRIPTION
**Pull request by katre**

### CODE REVIEW

File: _posts/2019-08-27-bazel-0.29.0.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+*   The [docker sandbox](https://docs.bazel.build/versions/master/remote-execution-sandbox.html) and the [execution log](https://docs.bazel.build/versions/master/remote-execution-caching-debug.html) now respect `--remote_default_platform_properties`.
+*   The [platform](https://docs.bazel.build/versions/master/be/platform.html#platform) attribute `remote_execution_properties` has been deprecated in favor of the newly introduced `exec_properties` attribute.
+
+
+## Java
+
+*   We fixed remote execution on Windows for the [Java rules](https://docs.bazel.build/versions/master/be/java.html). See [#8614](https://github.com/bazelbuild/bazel/issues/8614).
+*   You can now use a new [`java_toolchain`](https://docs.bazel.build/versions/master/be/java.html#java_toolchain) target `@remote_java_tools_{linux/windows/darwin}//:prebuilt_toolchain` in your builds. The new toolchain uses only pre-built tools, including singlejar and ijar, even on remote execution. The prebuilt toolchain  should be used only when host and execution platform are the same, otherwise the binaries will not work on the execution platform.
```
Comment (laszlocsomor) [inline _posts/2019-08-27-bazel-0.29.0.md]: > @remote_java_tools_{linux/windows/darwin}//:prebuilt_toolchain

This is neither valid Bash nor valid Label syntax.
Could you write maybe `@remote_java_tools_<PLATFORM>//:prebuilt_toolchain` and specify the values for that?

Like so:
> `@remote_java_tools_<PLATFORM>//:prebuilt_toolchain` (where `<PLATFORM>` is one of `linux`, `windows`, or `darwin`

File: _posts/2019-08-27-bazel-0.29.0.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+*   Added `--incompatible_load_python_rules_from_bzl`, which will be flipped in a future Bazel version. To migrate, load Python rules from `@rules_python`. See [#9006](https://github.com/bazelbuild/bazel/issues/9006).
+
+
+## Other
+
+*   Add new incompatible flag `--incompatible_load_proto_rules_from_bzl` to forbid loading the native proto rules. For migration instructions, see tracking issue [#8922](https://github.com/bazelbuild/bazel/issues/8922)
+*   `rule_test`: fix Bazel 0.27 regression ("tags" attribute was ignored, [#8723](https://github.com/bazelbuild/bazel/issues/8723))
+*   The `objc_proto_library` rule has been [deleted](https://github.com/bazelbuild/bazel/issues/7348) from Bazel.
+*   `@build_tools/tools/build_defs/pkg:{pkg_deb, pkg_rpm, pkg_tar}` are documented as deprecated. These features are now available independently of Bazel releases on the [`rules_pkg` repository](https://github.com/bazelbuild/rules_pkg/tree/master/pkg). The original rules will continue to exist until Bazel 2.0.
```
Comment (laszlocsomor) [inline _posts/2019-08-27-bazel-0.29.0.md]: > `@build_tools/tools/build_defs/pkg:{pkg_deb, pkg_rpm, pkg_tar}` are ...

This is neither valid Bash nor valid Label syntax. How about:

> `@build_tools/tools/build_defs/pkg:pkg_deb`, `:pkg_rpm`, and `:pkg_tar` are ...

File: _posts/2019-08-27-bazel-0.29.0.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+*   Added a new flag `--//tools/build_defs/pkg:incompatible_no_build_defs_pkg` to forbid loading rules from `//tools/build_defs/pkg:{pkg_deb, pkg_rpm, pkg_tar}`.
```
Comment (laszlocsomor) [inline _posts/2019-08-27-bazel-0.29.0.md]: @aiuto : Please verify the flag correct. Did you mean `--//tools/build_defs/pkg:incompatible_no_build_defs_pkg` or `--incompatible_no_build_defs_pkg`?

Comment (laszlocsomor) [reply _posts/2019-08-27-bazel-0.29.0.md]: Also, the `@build_tools/tools/build_defs/pkg:{pkg_deb, pkg_rpm, pkg_tar}` are not valid, I suggest writing it as:

> `@build_tools/tools/build_defs/pkg:pkg_deb`, `:pkg_rpm`, and `:pkg_tar`

Comment (laszlocsomor) [reply _posts/2019-08-27-bazel-0.29.0.md]: Also, does this flag have any effect outside of the Bazel source tree? (Does it require that `//tools/build_defs/pkg` be a valid package in the source tree?)

**Comment by laszlocsomor** [comment]
LGTM, module the incompat. flag question to Tony.

**Comment by laurentlb** [comment]
If it's not too late, you can add this section for the community updates.

```
## Community

    *   Li Haoyi from Databricks wrote about [Fast Parallel Testing at Databricks with Bazel](https://databricks.com/blog/2019/07/23/fast-parallel-testing-at-databricks-with-bazel.html) on their journey on reducing PR wait times for CI from ~3 hours with SBT to ~40 minutes with Bazel.
    *   Alex Eagle recorded screencasts on writing Bazel rules for [Stylus](http://stylus-lang.com/) ([tweet](https://twitter.com/Jakeherringbone/status/1149436749833105408), [screencasts](https://drive.google.com/corp/drive/folders/1I71w8JSGcZk1nUX2oXvT8NLT47Onb1t0))
    *   Alex Eagle talked about Bazel on the [Adventures in Angular 247](https://devchat.tv/adv-in-angular/aia-247-bazel-with-alex-eagle/) podcast.
    *   Vamsi Vempati wrote [Angular 8 - Bazel Walkthrough](https://blog.bitsrc.io/angular-8-bazel-walkthrough-f7585bcaf282).
    *   Wassim Chegham talked about [Bazel for Angular developers](https://www.youtube.com/watch?v=1BrNSFQo1rM) at Angular In Depth 2019.
```

**Comment by laszlocsomor** [comment]
If it's not too late, i want to add relnotes from https://github.com/bazelbuild/bazel/commit/d3bacfba3efef386bba80d7ff16c12c486165c4e:

RELNOTES[NEW]: genrule now supports `cmd_bash`, `cmd_ps`, `cmd_bat` attributes. Find more details from https://docs.bazel.build/versions/master/be/general.html#genrule.cmd

**Pull request merged**